functions/fuel.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+var mpg = 30;
+var distance = 204;
+var fuelCalc = function (){
```

Comment:
Again, the function needs to take parameter, one for the mpg figure, one for the distance.
Also, think of a better name,.